commit 777bde297bd5e6b0f4f358ef99a02ef34d8e4f5a
Author: Felipe Balbi <felipe.balbi@linux.intel.com>
Date:   Thu Jan 19 13:38:42 2017 +0200

    usb: dwc3: gadget: skip Set/Clear Halt when invalid
    
    commit ffb80fc672c3a7b6afd0cefcb1524fb99917b2f3 upstream.
    
    At least macOS seems to be sending
    ClearFeature(ENDPOINT_HALT) to endpoints which
    aren't Halted. This makes DWC3's CLEARSTALL command
    time out which causes several issues for the driver.
    
    Instead, let's just return 0 and bail out early.
    
    Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1309_777bde297bd5e6b0f4f358ef99a02ef34d8e4f5a.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1309_777bde297bd5e6b0f4f358ef99a02ef34d8e4f5a.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
